### ISSUE TITLE
Use nodescripts to reduce the footprint of any v2 stuff

### ISSUE THREAD

**Issue by drieJAC** [jac-uk/digital-platform#1130]
## Background
There are several pain points in the code base and database where we are using v1 and v2 to manage changes in data structure/processes. These are difficult to manage and will be a growing source of pain as we move forward.
We could look to use the following approaches:

- use nodescripts to migrate data from v1 to v2
- set dates when certain v1 versions become redundant so that the code no longer needs to support it and we can remove versioning in those specific areas of the code/db

The following are areas where we use v1/v2 stuff:

- working preferences

## User Story
As a developer I would like to remove versioning where possible so that we dont need to support multiple versions of the code

## Definition of Done
- [ ] User stories/acceptance criteria met
- [ ] Internal reviews passed (feedback actioned)
- [ ] User testing passed
- [ ] Relevant technical documentation updated
- [ ] User guidance updated
- [ ] Deployed and merged without errors

## User Testing Steps
Specify for users what is being tested (but not how to test it.)

**Comment by drieJAC** [comment]
@nickaddy Initially we will need discussions to work out where this is feasible and how we can do it. We can then write separate tickets for it.

**Comment by nickaddy** [comment]
Hey team! Please [add your planning poker estimate](https://app.zenhub.com/workspaces/Platform-Development-5ea838cd2aec471eb6d14139/issues/jac-uk/digital-platform/1130?planning-poker) with Zenhub @drieJAC @HalcyonJAC @KoWeiJAC @tomlovesgithub

**Comment by HalcyonJAC** [comment]
Please [add your planning poker estimate](https://app.zenhub.com/workspaces/Platform-Development-5ea838cd2aec471eb6d14139/issues/jac-uk/digital-platform/1130?planning-poker) with Zenhub @warrensearle

**Comment by warrensearle** [comment]
I don't think we can estimate this and feel the effort might be better spent, initially at least, on archiving and documenting our data model
